### PR TITLE
Add reporting to the docker composition, component maps and git versions

### PR DESCRIPTION
**Pull request by tranchitella**
Changelog: none

Signed-off-by: Fabio Tranchitella <fabio.tranchitella@northern.tech>

Depends on: https://github.com/mendersoftware/gui/pull/1885

### CODE REVIEW

**Comment by tranchitella** [comment]
@mender-test-bot start pipeline

**Comment by tranchitella** [comment]
@mender-test-bot start pipeline

**Review by lluiscampos — APPROVED** [review]
`component-maps.yml` and `release_tool.py` parts look good :+1:

File: docker-compose.reporting.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+version: '2.1'
+services:
+    #
+    # mender-reporting
+    #
+    mender-reporting:
+        image: mendersoftware/reporting:mender-master
+        command: server
```
Comment (mchalski) [inline docker-compose.reporting.yml:8]: just need `server --automigrate` here.

the basic migration will then insert the index template, so the index can be auto created when tenant data comes in.

(without it we won't get our special field mappings)

**Review by mchalski — APPROVED** [review]

**Comment by tranchitella** [comment]
@mender-test-bot start pipeline

**Pull request merged**